### DISCUSSION TITLE
Question about "segfault at 7f377e872428 (sp 00007ffcd580fec0 error 4)"

### DISCUSSION THREAD

**Discussion by drsheep404** [oerdnj/deb.sury.org#2135] [General]
Hi,

I want ask and get sure about the issue here (https://github.com/php/php-src/issues/14261) - can it be related to this repository (packages)?
Because the answer of this (https://github.com/php/php-src/issues/14261#issuecomment-2118820576) may be a key, how this seqfault happen.

We use this repository for Debian 12 (bookworm), latest version:
`deb https://packages.sury.org/php/ bookworm main
`
Thanks and best regards! :)

**Comment by oerdnj** [comment]
There are no custom patches for opcache, so I doubt it’s related to the packaging.
